serverlessworkflow/modules/ROOT/pages/cloud/operator/sonataflow-metrics.adoc

```diff
@@ -0,0 +1,99 @@
+= SonataFlow Metrics
```

Comment:
I think this page is not shown in the menu Cloud -> Operator or in any other menu entry.

On the other hand, we have a sort of metrics page that shows the metrics and that I personally don't like 100%.
I think that what we must do, is to do a kind of merge between this metrics content and what is shown in the page below, and provide something good.
But , it's out of from @jianrongzhang89 scope.

see below:
![image](https://github.com/user-attachments/assets/afad22bd-9444-4553-81c9-52e44e24b372)

Feel free to merge as is, to not loose this content and we can restructure in a followup PR.
@ricardozanini   @domhanak

Reply:
@ricardozanini @wmedvede @domhanak I tried to merge them. Please review.